build/styleguide.config.js

```diff
@@ -11,21 +12,20 @@ const extractPlugin = ExtractTextPlugin.extract({
 module.exports = {
   webpackConfig: Object.assign(
     {},
-    require('./webpack.dev.config.js'),{
+    require('./webpack.prod.config.js'),{
       devServer: {
 	contentBase: resolve('../docs'),
-	publicPath: '/dev/',
+	publicPath: '/docs/',
 	disableHostCheck: true
       }
     }),
     mixins: [
-      '../src/mixins/styleguide/styleguidedist.js'
+	'../src/mixins/styleguide/styleguidist.js'
```

Comment:
Is this purposefully spelled wrong?

Reply:
https://github.com/vue-styleguidist/vue-styleguidist 🤷‍♂️